rust/db_handling/src/interface_signer.rs

```diff
@@ -160,7 +162,12 @@ where
         .map(|(multisigner, address_details)| {
             let address_key = AddressKey::from_multisigner(&multisigner); // to click
             let public_key = multisigner_to_public(&multisigner); // to display
-            let identicon = make_identicon_from_multisigner(&multisigner);
+            let style = if address_details.encryption == Encryption::Ethereum {
+                IdenticonStyle::Blockies
+            } else {
+                IdenticonStyle::Dots
+            };
```

Comment:
As we repeat this `if` multiple times in this PR, wouldn't it be better to wrap it in method that would return `IndenticonStyle` based on input `Encryption`, like:

```rust
fn identicon_style(encryption: Encryption) -> IdenticonStyle 
```
This could also be useful if there are any new styles
Or `make_identicon_from_multisigner` could take additional `Encryption` parameter and resolve style internally 🤔